code/style.css

```diff
@@ -1,58 +1,176 @@
+/*** GENERAL STYLING ***/
+
 body {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  font-family: 'Roboto', sans-serif;
+  background: url(./assets/kamil-feczko-GhxWry42_zQ-unsplash.jpg);
```

Comment:
It's far more safe approach to wrap path in parentheses. Without them, some browsers might not find the path
`url("./assets/kamil-feczko-GhxWry42_zQ-unsplash.jpg");`